kuksa-client/README.md

```diff
@@ -55,36 +50,51 @@ Welcome to Kuksa Client version <some_version>
 
 Default tokens directory: /some/path/kuksa_certificates/jwt
 
-connectj to wss://127.0.0.1:8090
-Websocket connected securely.
+Connecting to VSS server at 127.0.0.1 port 55555 using KUKSA GRPC protocol.
+TLS will not be used.
+INFO 2023-09-15 18:48:13,415 kuksa_client.grpc No Root CA present, it will not be posible to use a secure connection!
+INFO 2023-09-15 18:48:13,415 kuksa_client.grpc.aio Establishing insecure channel
+gRPC channel connected.
 Test Client>
 ```
 
-If the connected KUKSA Server or KUKSA Databroker require authorization the next step is to authorize.
-KUKSA Server and KUKSA Databroker use different token formats.
+If you wish to connect to a VISS server e.g. `kuksa-val-server` (not using TLS), you should instead run:
+
+```console
+$ kuksa-client ws://127.0.0.1:8090
+```
+
+### TLS with databroker
+
+KUKSA Client use TLS when the schema part of the server URI is `grpcs` or `wss`, i.e. a valid uri to connect to a TLS enabled local databroker is
+
+```
+grpcs://127.0.0.1:55555
+```
+
+To contact a TLS enabled local val-server the uri is
 
-### Connecting to KUKSA Databroker
+```
+wss://127.0.0.1:8090
+```
 
-A gRPC connection to KUKSA Databroker is started by specifying address and port for the Databroker and giving
-`--protocol grpc` as argument.
-KUKSA Client use TLS by default, it only run in insecure mode if `--insecure` is given as argument.
-By default the KUKSA example Root CA and Client keys are used, but client keys have no effect as mutual authentication is not supported by KUKSA Databroker or KUKSA Server.
+By default the KUKSA example Root CA and Client keys are used, but client keys have no effect currently as mutual authentication is not supported by KUKSA Databroker or KUKSA Server.
 
 ```
-~/kuksa.val/kuksa-client$ kuksa-client --ip localhost --port 55555 --protocol grpc
+~/kuksa.val/kuksa-client$ kuksa-client grpcs://localhost:55555
 ```
 
 This call with all parameters specified give same effect:
 
 ```
-~/kuksa.val/kuksa-client$ kuksa-client --ip localhost --port 55555 --protocol grpc --certificate ../kuksa_certificates/Client.pem --keyfile ../kuksa_certificates/Client.key --cacertificate ./kuksa_certificates/CA.pem
+~/kuksa.val/kuksa-client$ kuksa-client --certificate ../kuksa_certificates/Client.pem --keyfile ../kuksa_certificates/Client.key --cacertificate ./kuksa_certificates/CA.pem grpcs://localhost:55555
 ```
 
 There is actually no reason to specify client key and certificate, as mutual authentication is not supported in KUKSA Databroker,
 so the command can be simplified like this:
 
 ```
-~/kuksa.val/kuksa-client$ kuksa-client --ip localhost --port 55555 --protocol grpc --cacertificate ./kuksa_certificates/CA.pem
+~/kuksa.val/kuksa-client$ kuksa-client  --cacertificate ./kuksa_certificates/CA.pem grpcs://localhost:55555
```

Comment:
```suggestion
~/kuksa.val/kuksa-client$ kuksa-client  --cacertificate ./kuksa_certificates/CA.pem --tls-server-name server grpcs://localhost:55555
```
otherwise it fails but I do not know if we want to add :)

Reply:
And line 105 below must be changed to new syntax

Reply:
Ah yes with localhost it works seems to be this problem :)